pandas/tests/generic/test_frame.py

```diff
@@ -268,3 +268,15 @@ def test_deepcopy_empty(self):
         empty_frame_copy = deepcopy(empty_frame)
 
         self._compare(empty_frame_copy, empty_frame)
+
+    def test_nested_dict_construction(self):
+        columns = ['Nevada', 'Ohio']
+        pop = {'Nevada': {2001: 2.4, 2002: 2.9},
```

Comment:
can you move to pandas/tests/frame/test_construction instead

Reply:
done